src/vs/workbench/contrib/debug/browser/debugEditorActions.ts

```diff
@@ -133,6 +134,33 @@ class LogPointAction extends EditorAction2 {
 	}
 }
 
+class GoToDisassemblyAction extends EditorAction {
+
+	public static readonly ID = 'editor.debug.action.goToDisassembly';
+	public static readonly LABEL = nls.localize('goToDisassembly', "Go to Disassembly");
```

Comment:
I suggest to call this action
"Open Disassembly View"

That way it is more aligned with our other commands that open views.
"Go to"  is not fitting here since the user is not going anywhere :)

fyi @weinand

Reply:
Yes, "Open *assembly View" is preferred.

Name discussion: "Disassembly, Assembly or Memory":

If I understand the Disassembly and the Memory features correctly, then there will be two separate viewers, one for showing disassembled instructions (from memory) and one for showing hexadecimal data (from memory), correct?
It theory there could be a single "Memory view" with two modes: a disassembly and a data mode. But I don't think that this is planned?

"Disassembly vs. Assembly":

I agree with Isidor that "Assembly view" appears to be simpler: the viewer shows "assembly code".

The only problem I see with that name is that users could expect this to be an assembly editor where they can make assembly source changes and execute them.

Calling it "Disassembly view" makes it more clear that this view only show some "disassembled data" and that it is not an "assembly editor".

Reply:
As mentioned in other comment, ultimately, our goal is to achieve similar functions as in VS. I.e., when right clicking a certain line in source (may not be %rip), go to disassembly corresponds to that line. This is why I think "go to disassembly" makes sense.

Also, this name is consistent with that in VS.

Reply:
@xisui-MSFT please note that name consistency with VS is not a goal of VS Code.
"Go to" has a specific semantics in VS Code, and "Go to Disassembly" is not a good match for that.
Please see the commands in VS Code's "Go" menu and review commands starting with "Go to" in the "Command Palette".

Reply:
Thanks @xisui-MSFT for providing more details about the motivation.
As I mentioned in https://github.com/microsoft/vscode/issues/124163#issuecomment-857579113

"And in general I would suggest to take a step back and think about how the experience would fit in VS Code the best. It can of course be inspired by the Visual Studio experience but try not to do a one for one copy. What was bad about the Visual Studio experience, can we somehow improve it this time around?"

With this in mind I would strongly suggest we use a more user friendly name that fits the overall naming patterns of VS Code.

Reply:
Sounds good to me. I'll change the name.